syft/pkg/cataloger/golang/version.go

```diff
@@ -1,226 +1,99 @@
 // This code was copied from the Go std library.
```

Comment:
this statement is no longer true -- before this PR it was true. It seems like this file has gone through a rewrite entirely. Also, the name `version.go` came from the standard lib as well; is there a better name for this now?

Reply:
removed, thx